### PR TITLE
e2e: upgrades Cypress to 4.50

### LINKED ISSUE #24031: Upgrade @grafana/e2e to latest cypress

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
Had to split up the biggest test into several it functions instead as performance after the upgrade was terrible as noted by Torkel in https://github.com/grafana/grafana/pull/22795.

**Which issue(s) this PR fixes**:
Fixes #24031

**Special notes for your reviewer**:

### CODE REVIEW

File: e2e/suite1/specs/queryVariableCrud.spec.ts [hunk 1]
```diff
@@ -1,74 +1,173 @@
 import { e2e } from '@grafana/e2e';
+import { describe } from '../../../public/test/lib/common';
```
Comment (mckn) [inline e2e/suite1/specs/queryVariableCrud.spec.ts:2]: question: do we want to have a dependency in this direction? from this package on to the grafana project?

Comment (hugohaggmark) [reply e2e/suite1/specs/queryVariableCrud.spec.ts:2]: > question: do we want to have a dependency in this direction? from this package on to the grafana project?

Exceptional catch, no we do not. I will fix! 🏅

File: e2e/suite1/specs/queryVariableCrud.spec.ts [hunk 2]
```diff
@@ -1,74 +1,172 @@
 import { e2e } from '@grafana/e2e';
+import { Flows } from '@grafana/e2e/src/flows';
```
Comment (dprokop) [inline e2e/suite1/specs/queryVariableCrud.spec.ts:2]: Shouldn't Flows be exported from e2e? Not saying no to path imports, but wondering if consumer shouldn't have access to Flows.

Comment (hugohaggmark) [reply e2e/suite1/specs/queryVariableCrud.spec.ts:2]: Great suggestion, I'll do that!

Comment (hugohaggmark) [reply e2e/suite1/specs/queryVariableCrud.spec.ts:2]: Aaah, I forgot I should use `e2e.flows` instead.

Comment (dprokop) [reply e2e/suite1/specs/queryVariableCrud.spec.ts]: perfect!

**Review by mckn — APPROVED** [review]
I think it looks good! Great job @hugohaggmark

**Review by dprokop — APPROVED** [review]

**Pull request merged**